File: codigofonte6 (terminar tradução)/1/positivo.py
# Abstração e escopo

from cs50 import get_int


def main():
    i = receber_inteiro_positivo()
    print(i)


# Pede ao usuário para informar um inteiro positivo
def receber_inteiro_positivo():
    while True:
        n = get_int("Inteiro positivo: ")
        if n > 0:
            break
    return n


main()

File: codigofonte6 (terminar tradução)/1/positivo.c
// Abstração e escopo

#include <cs50.h>
#include <stdio.h>

int receber_inteiro_positivo(void);

int main(void)
{
    int i = receber_inteiro_positivo();
    printf("%i\n", i);
}

// Pede ao usuário para informar um inteiro positivo
int receber_inteiro_positivo(void)
{
    int n;
    do
    {
        n = get_int("Inteiro positivo: ");
    }
    while (n < 1);
    return n;
}
